package.json

```diff
@@ -68,16 +68,16 @@
     "xpath": "~0.0.32"
   },
   "scripts": {
-    "coverage:test:core": "nyc npm run test:core",
-    "coverage:test:frontend": "nyc --nycrc-path .nycrc-frontend.json npm run test:frontend",
-    "coverage:test:package": "nyc npm run test:package",
-    "coverage:test:entrypoint": "nyc npm run test:entrypoint",
-    "coverage:test:integration": "nyc npm run test:integration",
-    "coverage:test:services": "nyc npm run test:services",
+    "coverage:test:core": "c8 npm run test:core",
+    "coverage:test:frontend": "c8 -c .nycrc-frontend.json npm run test:frontend",
+    "coverage:test:package": "c8 npm run test:package",
+    "coverage:test:entrypoint": "c8 npm run test:entrypoint",
+    "coverage:test:integration": "c8 npm run test:integration",
+    "coverage:test:services": "c8 npm run test:services",
     "coverage:clean": "rimraf .nyc_output coverage",
```

Comment:
Same theme here relative to a trial period for c8. Plus this directory will likely already exist on the local fs for regular contributors so doesn't hurt to keep it in here for a little while regardless